CIScripts/Test/PesterLogger/PesterLogger.ps1

```diff
@@ -21,25 +21,57 @@ function Initialize-PesterLogger {
     $ConstOutdir = Resolve-Path $Outdir
 
     $WriteLogFunc = {
-        Param([Parameter(Mandatory = $true)] [object] $Message)
+        Param(
+            [Parameter(Mandatory = $true)] [object] $Message,
+            [parameter(ValueFromRemainingArguments=$true)] $WriterArgs,
+            [Parameter(Mandatory=$false)] [string] $Tag = "tester",
```

Comment:
I don't think this name would actually lead to confusion, especially when testers' hostnames are `winci-tester-x`, not just `tester`. I think of _tester_ as a just machine performing tests -- be it ci tester, laptop or something else.

If you insist of changing this default, what do you think about:
* Defaulting to hostname
* Not printing a label at all for default messages
* Changing the default to `pester`
* Changing the default to `root` (I can see this will be misleading and provoke a questions "what is a root?")

Changing the default to `default` is a bad idea imho as this name is totally meaningless for somebody unaware of how our logging is implemented.

> MKos: What do you think about adding specialized Host-LogSource?

I think this would make sense when two-stage `Collect` + `Merge-Logs` will be implemented, which we've agreed to be out of scope of this PR.